skiko/build.gradle.kts

```diff
@@ -380,7 +381,10 @@ tasks.withType(LinkSharedLibrary::class.java).configureEach {
                     "Dwmapi.lib",
                     "opengl32.lib",
                     "shcore.lib",
-                    "user32.lib"
+                    "user32.lib",
+                    "d3d12.lib",
```

Comment:
Does it mean we must have dx12 dlls on the target system?

Reply:
As far as I understand, the new graphics card drivers contain the d3d12 library, but depending on the OS and `D3D_FEATURE_LEVEL_11_0` may create a DirectX12 or DirectX11 context. But I cannot verify this yet. The best solution is to test this on Windows 8 / 8.1 because Microsoft has released DirectX12 support on Windows 7.